Dockerfile.bats

```diff
@@ -34,5 +41,11 @@ COPY --from=builder /go/src/github.com/openshift/secrets-store-csi-driver/google
 RUN ln -s /usr/local/google-cloud-sdk/bin/gcloud /usr/local/bin/gcloud
 RUN gcloud version
 
+# Copy Azure CLI
+COPY --from=builder /usr/bin/az /usr/bin/az
+COPY --from=builder /usr/lib64/az /usr/lib64/az
+
 # Install envsubst and less
-RUN dnf install -y gettext less && dnf clean all
+RUN dnf install -y python3.12 gettext less && dnf clean all
```

Comment:
Wondering why `python3.12` installation is required for the base image and not for the builder image.